website/docs/r/federated_query_limit.html.markdown

```diff
@@ -0,0 +1,60 @@
+---
+layout: "mongodbatlas"
+page_title: "MongoDB Atlas: federated_database_query_limit"
+sidebar_current: "docs-mongodbatlas-resource-federated-query-limit"
+description: |-
+    Provides a Federated Database Instance Query Limit.
+---
+
+# Resource: mongodbatlas_federated_query_limit
+
+`mongodbatlas_federated_query_limit` provides a Federated Database Instance Query Limits data source.
+
+
+-> **NOTE:** Groups and projects are synonymous terms. You may find group_id in the official documentation.
+
+## Example Usages
+
+
+```terraform
+resource "mongodbatlas_federated_query_limit" "test" {
```

Comment:
I personally prefer examples that have real values.   We've done that in most of our docs up to now. I'd prefer to keep them real world examples, so there would be a limit_name, overrun_policy and value.

Reply:
updated

Reply:
TY!